include/crm/common/alerts_internal.h

```diff
@@ -63,4 +74,8 @@ void crm_set_alert_key_int(enum crm_alert_keys_e name, int value);
 void crm_unset_alert_keys(void);
 void crm_set_envvar_list(crm_alert_entry_t *entry);
 void crm_unset_envvar_list(crm_alert_entry_t *entry);
+lrmd_key_value_t * crm_set_alert_key_to_lrmd_params(lrmd_key_value_t *head, enum crm_alert_keys_e name, const char *value);
+void crm_set_alert_envvar_to_lrmd_params(crm_alert_entry_t *entry, lrmd_key_value_t * params);
```

Comment:
Yes, lib/lrmd/lrmd_alerts.c is good. The header file should be under include/crm, though; common/ is for libcrmcommon. (Ideally, we'd have an lrmd/ directory for lrmd.h and lrmd_alert_internal.h, but lrmd.h is part of the public API, so it's too late to move it now.)